conbench/templates/benchmark-result.html

```diff
@@ -206,7 +206,16 @@ <h4>raw data</h4>
               {{ k }}
               {% if v is not none %}
                 <div align="right" style="display:inline-block; float: right;">
-                  <code>{{ v | urlize(target="_blank") }}</code>
+                  {% if v is iterable and (v is not string and v is not mapping) %}
+                    <code>[
+                      {% for i in v %}
+                        {{ i | urlize(target="_blank") }}
+                        {% if not loop.last %},{% endif %}
```

Comment:
I'm fine with this. But would you rather put a newline between elements than a comma? Up to you!

Reply:
If the list always contained links, that would be quite nice indeed, but if the list contains other things it may appear rather weird. For this reason I tried to keep the default way of rendering lists as much as possible but still have the clickable URLs.